src/components/Nav/Nav.jsx

```diff
@@ -30,14 +35,21 @@ function Nav() {
             <Link className="navLink" to="/admin">
               Admin
             </Link>
+        {/* will be the Admin link for the companies page */}
+            {/* <Link className="navLink" to="/SearchPage">
+              Companies
+            </Link> */}
+        {/* will be the Admin link for the feedback page */}
+            <Link className="navLink" to="/info">
+              Feedback
```

Comment:
this is currently Admin ( /admin) on line 35.  We can merge this into master then combine them.. probably better to name the /admin as Feedback